frontend/packages/dev-console/src/components/import/devfile/DevfileImportForm.tsx

```diff
@@ -33,11 +65,31 @@ const DevfileImportForm: React.FC<FormikProps<FormikValues> & DevfileImportFormP
             {devfileParseError}
           </Alert>
         )}
-        <GitSection
-          buildStrategy="Devfile"
-          builderImages={builderImages}
-          defaultSample={{ url: 'https://github.com/redhat-developer/devfile-sample' }}
-        />
+        {formType === 'sample' ? (
+          <FormSection>
+            <InputField
+              type={TextInputTypes.text}
+              name="git.url"
+              label={t('devconsole~Git repo URL')}
+              data-test-id="git-form-input-url"
+              isDisabled
+            />
+            {selectedBuilderImage && (
+              <ImageStreamInfo
+                displayName={selectedBuilderImage.name}
+                tag={selectedBuilderImage.tags[0]}
+              />
+            )}
+          </FormSection>
+        ) : (
+          <GitSection
```

Comment:
There's one more flow that this form should handle. The dev catalog flow where user comes from dev catalog, in that case then git URL will be filled but the inout field will not be disabled.

Reply:
Updated.